Add ERC: Security token interface for confidentiality and legal status transparency to crypto security tokens (#143)
ethereum/ERCs: ERCS/erc-xxx.md

> ---

Reviewer: ```suggestion
---
eip: 7570
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

> title: Smart Contract Interface to implement confidentiality and legal status transparency to crypto security tokens

Reviewer: ```suggestion
title: Security token interface for confidentiality and legal status transparency
```
Suggest a shorter title
